install/roles/certificates-all/defaults/main.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
I think we can remove this file, it is an artefact of a first attempt to remove the certificates variable initialisation from the main playbook.